minishift-ci-index.yaml

```diff
@@ -8,6 +8,10 @@
         - ansicolor
         - workspace-cleanup
         - timestamps
+        - credentials-binding:
+            - text:
+                credential-id: 1a087ac4-4a16-4d26-99f5-197a9ec00c6e
```

Comment:
This isn't the access token, this is an index to the Jenkins credential store where the token is actually located.